rust/arrow/src/array/builder.rs

```diff
@@ -2031,6 +2059,7 @@ where
     /// Append a primitive value to the array. Return an existing index
     /// if already present in the values array or a new index if the
     /// value is appended to the values array.
+    #[inline]
```

Comment:
This seems a large function to inline? Maybe we need to be conservative here and not mark it as inline

Reply:
Ok, I will remove that one.